__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const request = require('supertest')
+const app = require('../app/app')
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+    test('200: should respond with an array of all topic objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body}) => {
+            const  { topics} = body;
```

Comment:
A good test so far, though this test should also check the length of the topics array, because if the topics array was empty, we would get a incorrectly passing test, as this loop would be skipped over.
Could you please add an expect statement which checks the length of the array

Reply:
okay. great. thanks. will implement the length check